src/registrar/admin.py

```diff
@@ -4,13 +4,32 @@
 from django.contrib.contenttypes.models import ContentType
 from django.http.response import HttpResponseRedirect
 from django.urls import reverse
+from registrar.models.utility.admin_form_order_helper import AdminFormOrderHelper, SortingDictInterface # noqa
 from . import models
 
 logger = logging.getLogger(__name__)
+# The linter does not like the length of SortingDictInterface, so these are split here
+audited_admin_item_names = ["submitter", "authorizing_official",
+                            "investigator", "creator", "user"]
+audited_admin_orderby_names = ['first_name', 'last_name']
+
+contact_admin_item_names = ["domain", "requested_domain"]
+contact_admin_orderby_names = ["name"]
+# Used to keep track of how we want to order_by certain FKs
+foreignkey_orderby_dict: [SortingDictInterface] = [
+    # foreign_key - order_by
+    SortingDictInterface(
+                        audited_admin_item_names,
+                        audited_admin_orderby_names
+                        ).sorting_dict,
+    SortingDictInterface(
+                        contact_admin_item_names,
+                        contact_admin_orderby_names
+                        ).sorting_dict
+]
 
 
 class AuditedAdmin(admin.ModelAdmin):
```

Comment:
Suggest keeping AuditedAdmin a one job class, making a new SortFields class and have one inherit from the other.

Reply:
Yes I agree with this

Reply:
@abroddrick @rachidatecs Got that changed for you guys